### ISSUE TITLE
Passport doesn't redirect back to authorization form after login

### ISSUE THREAD

**Issue by BrandOriented** [laravel/passport#703]
Problem related to https://github.com/laravel/passport/issues/248

Problem only on chrome(tested from 64-67).
If I enter through the link directly in the new tab, the problem does not occur, but if it is in the same card, laravel gets an invalid url()->previous()

**Comment by BrandOriented** [comment]
This problem occurs only local domains eg. (loc, tld)

**Comment by ingria** [comment]
No, it happens on any domain. It doesn't happen in mobile browsers for some reason.

**Comment by driesvints** [comment]
I see that the original issue uses Passport 1.0, can you confirm that this is happening on the latest Passport release as well?

Feel free to reply if this still happens with the latest Laravel and Passport release and I'll have a look.

**Comment by BrandOriented** [comment]
@driesvints Problem occurs on 7.x version

**Comment by driesvints** [comment]
I'll try to check in on this when I find some time.

**Comment by driesvints** [comment]
I'll need more info and/or code to debug this further. Please post relevant code like models, jobs, commands, notifications, events, listeners, controller methods, routes, etc. You may use https://paste.laravel.io to post larger snippets or just reply with shorter code snippets. Thanks!

**Comment by driesvints** [comment]
Closing this issue because it's inactive, already solved, old or not relevant anymore. Feel free to reply if you're still experiencing this issue.

**Comment by satyakresna** [comment]
Hi, I just solve the problem and provide the gist below:

[Fixed problem Laravel Passport doesn't redirect back to authorization form after login](https://gist.github.com/satyakresna/5031ee18b25e317341c52bc6a7c39671)

I hope it helps you all.

**Comment by pogush** [comment]
I had this very same issue while trying Passport with Laravel 8 and Breeze.

Fixed by changing the `store(LoginRequest $request)` function in `AuthenticatedSessionController` to return

`return redirect()->intended(RouteServiceProvider::HOME)`

I also made the same change in the `RedirectIfAuthenticated`.

This seems more like a Breeze issue/enhancement by the way, but figured it could help someone else if I posted this here.
